app/controllers/application_controller.rb

```diff
@@ -1,4 +1,16 @@
 # frozen_string_literal: true
 class ApplicationController < ActionController::Base
   protect_from_forgery with: :exception
+
+  def append_info_to_payload(payload)
```

Comment:
This should be add `Decidim` too, since Decidim's application controller won't inherit/use this controller. We can merge this and enhance it later from Decidim.